app/game/game.js

```diff
@@ -1,8 +1,9 @@
+// @ts-check
```

Comment:
typescript??

Reply:
That's just a little check to have vs code help us catch errors or bugs, try it out it may help you see bugs or errors before it actually runs